### DISCUSSION TITLE
Adding a device, MyDevice folder not shows up in the project of vtkPlusDataCollection project

### DISCUSSION THREAD

**Discussion by 1939938853** [PlusToolkit/PlusLib#934] [Q&A]
Hello,

I'm trying to add a device following the instruction[ here](https://plustoolkit.github.io/devicecode). But MyDevice folder not shows up in the project of vtkPlusDataCollection project. What can be the problem?

I followed every step in the[ instruction,](https://plustoolkit.github.io/devicecode) and enabled my device AimPosition at cmake config of PlusBuild.  Open the pluslib solution in pluslib-bin. My device folder PlusLib\src\PlusDataCollection\AimPosition, just not show up.
![Capture1](https://user-images.githubusercontent.com/59957922/163609118-cb1451f9-a887-444d-95e0-640b8641c71d.JPG)

**Comment by adamrankin** [comment]
Can you output the value of AimPosition_HDRS and AimPosition_SRCS and see if they have valid values?

```cmake
message("headers: ${AimPosition_HDRS}")
message("sources: ${AimPosition_SRCS}")
```

**Comment by 1939938853** [comment]
It seems that the selected PLUS_USE_MyDevice camke option at the superbuild cmake config of plusBuild can not be passed to vtkPlusDataCollection project.  Neither the defined variable in FindMyDevices.cmake at PlusBuild\Modules, can passed to vtkPlusDataCollection project.  How this can be solved? Thanks!
